datest/run.go

```diff
@@ -0,0 +1,165 @@
+package datest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/commonjava/indy-tests/common"
+)
+
+type Report struct {
+	ExecutionRoot struct {
+		GroupID     string `json:"groupId"`
+		ArtifactID  string `json:"artifactId"`
+		Version     string `json:"version"`
+		OriginalGAV string `json:"originalGAV"`
+	} `json:"executionRoot"`
+
+	Modules []struct {
+		Gav struct {
+			GroupID     string `json:"groupId"`
+			ArtifactID  string `json:"artifactId"`
+			Version     string `json:"version"`
+			OriginalGAV string `json:"originalGAV"`
+		} `json:"gav"`
+		ManagedPlugins struct {
+		} `json:"managedPlugins"`
+		ManagedDependencies struct {
+			Dependencies map[string]struct {
+				GroupID    string `json:"groupId"`
+				ArtifactID string `json:"artifactId"`
+				Version    string `json:"version"`
+			} `json:"dependencies"`
+		} `json:"managedDependencies,omitempty"`
+	} `json:"modules"`
+}
+
+func getAlignLog(url string) string {
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		panic(err)
+	}
+	req.Header.Set("Accept", "text/plain")
+
+	var c http.Client
+	resp, err := c.Do(req)
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	responseData, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		panic(err)
+	}
+
+	return string(responseData)
+}
+
+func lookupMetadata(url string) string {
```

Comment:
Will this "string" return be removed as no string result is returned here?

Reply:
Sure, will remove it. We can add it later if needed.